utils/deployment-utils/deployment_utils.go

```diff
@@ -189,9 +190,6 @@ func ConstructKeylessTransaction(
 	return contractCreationTxBytes, senderAddress, contractAddress, nil
 }
 
-// AlterNicksTransaction increments the gas price of the transaction used to deploy the contract.
-// This changes the derived contract and deployer address, which is useful for deploying the same
-// Teleporter contract to different addresses.
-func AlterNicksTransaction() {
-	contractCreationGasPrice.Add(contractCreationGasPrice, big.NewInt(1))
+func GetDefaultContractCreationGasPrice() *big.Int {
+	return defaultContractCreationGasPrice
```

Comment:
Can we copy the big.Int here? Returning the pointer to the global var means it can be altered